src/main/java/io/confluent/idesidecar/restapi/connections/DirectConnectionState.java

```diff
@@ -241,8 +224,8 @@ protected Future<KafkaClusterStatus> getKafkaConnectionStatus() {
   protected Future<SchemaRegistryStatus> getSchemaRegistryConnectionStatus() {
     return withSchemaRegistryClient(srClient -> {
       // There is a configuration, so validate the connection by creating a SchemaRegistryClient
-      // and getting all subjects.
-      srClient.getAllSubjects();
+      // and getting all schema types
+      srClient.getSchemaTypes();
```

Comment:
`getSchemaTypes` is much less expensive than a `getAllSubjects` call. This is kind of unrelated to the PR's changes.